Content.Client/GameObjects/EntitySystems/HotbarSystem.cs

```diff
@@ -0,0 +1,155 @@
+                new PointerInputCmdHandler((in PointerInputCmdArgs args) => { return HandleHotbarKeybindPressed(1, args); }));
+            inputSys.BindMap.BindFunction(ContentKeyFunctions.Hotbar2,
+                new PointerInputCmdHandler((in PointerInputCmdArgs args) => { return HandleHotbarKeybindPressed(2, args); }));
+            inputSys.BindMap.BindFunction(ContentKeyFunctions.Hotbar3,
+                new PointerInputCmdHandler((in PointerInputCmdArgs args) => { return HandleHotbarKeybindPressed(3, args); }));
+            inputSys.BindMap.BindFunction(ContentKeyFunctions.Hotbar4,
+                new PointerInputCmdHandler((in PointerInputCmdArgs args) => { return HandleHotbarKeybindPressed(4, args); }));
+            inputSys.BindMap.BindFunction(ContentKeyFunctions.Hotbar5,
+                new PointerInputCmdHandler((in PointerInputCmdArgs args) => { return HandleHotbarKeybindPressed(5, args); }));
+            inputSys.BindMap.BindFunction(ContentKeyFunctions.Hotbar6,
+                new PointerInputCmdHandler((in PointerInputCmdArgs args) => { return HandleHotbarKeybindPressed(6, args); }));
+            inputSys.BindMap.BindFunction(ContentKeyFunctions.Hotbar7,
+                new PointerInputCmdHandler((in PointerInputCmdArgs args) => { return HandleHotbarKeybindPressed(7, args); }));
+            inputSys.BindMap.BindFunction(ContentKeyFunctions.Hotbar8,
+                new PointerInputCmdHandler((in PointerInputCmdArgs args) => { return HandleHotbarKeybindPressed(8, args); }));
+            inputSys.BindMap.BindFunction(ContentKeyFunctions.Hotbar9,
+                new PointerInputCmdHandler((in PointerInputCmdArgs args) => { return HandleHotbarKeybindPressed(9, args); }));
+        }
+
+        public override void Shutdown()
+        {
+            base.Shutdown();
+
+            if (!EntitySystemManager.TryGetEntitySystem<InputSystem>(out var inputSys))
+            {
+                return;
+            }
+            inputSys.BindMap.UnbindFunction(ContentKeyFunctions.OpenAbilitiesMenu);
+            inputSys.BindMap.UnbindFunction(ContentKeyFunctions.Hotbar0);
+            inputSys.BindMap.UnbindFunction(ContentKeyFunctions.Hotbar1);
+            inputSys.BindMap.UnbindFunction(ContentKeyFunctions.Hotbar2);
+            inputSys.BindMap.UnbindFunction(ContentKeyFunctions.Hotbar3);
+            inputSys.BindMap.UnbindFunction(ContentKeyFunctions.Hotbar4);
+            inputSys.BindMap.UnbindFunction(ContentKeyFunctions.Hotbar5);
+            inputSys.BindMap.UnbindFunction(ContentKeyFunctions.Hotbar6);
+            inputSys.BindMap.UnbindFunction(ContentKeyFunctions.Hotbar7);
+            inputSys.BindMap.UnbindFunction(ContentKeyFunctions.Hotbar8);
+            inputSys.BindMap.UnbindFunction(ContentKeyFunctions.Hotbar9);
+        }
+
+        private void HandleOpenAbilitiesMenu()
+        {
+            var playerEnt = _playerManager.LocalPlayer.ControlledEntity;
+            if (playerEnt == null
```

Comment:
Does this check prevent actually occurring errors, or is it just-in-case?

Reply:
This is a copy paste from the HandsSystem modified from getting HandsComponent to HotbarComponent, so I assumed it was needed.